pkg/controller/operatorsource/operatorsource_controller.go

```diff
@@ -23,7 +24,7 @@ func Add(mgr manager.Manager) error {
 }
 
 // newReconciler returns a new reconcile.Reconciler
```

Comment:
This comment is out of sync with the function.

Reply:
This is also present in the `CatalogSourceConfig` code. I will fix it there too.